tests/baselines/reference/templateInsideCallback.js

```diff
@@ -146,7 +146,7 @@ declare function flatMap(): any;
  * @template T
  * @type {Call<T>}
  */
-declare const identity: any;
+declare const identity: Call<T>;

```

Comment:
There's a `Type Call is not generic` error on the input here, so the output is fairly flexible. Preserving the erroneous input node is fine, since it's what `isolatedDeclarations` wants.